packages/frontend/tests1/e2e/07-strategy.spec.js

```diff
@@ -0,0 +1,75 @@
+/// <reference types="cypress" />
+import TradePage from '../pages/trade'
```

Comment:
@yichiehliu Why this file is in a different location? Can we move it with other test fikes